src/parsers/Ensembl-Uniprot_map_generator.py

```diff
@@ -27,9 +27,12 @@
 
     ## Report input data:
     print('[Info] Report on the Ensembl mapping file:')
+    print('[Info] Dropping non-Ensembl gene IDs...')
+    ensembl_map_df = ensembl_map_df.loc[ensembl_map_df.ensembl_id.str.match('ENSG.+')]
```

Comment:
The gene id-s in the ensembl derived map file are not `ENSG\d+` for 3880 proteins:
```
grep -v "ENSG"  temp/parsed_tables/uniprot2ensembl_draft.tsv |  head
```
Gives:
```
ensembl_id
LRG_376
LRG_376
LRG_376
LRG_571
LRG_571
LRG_571
LRG_777
LRG_777
LRG_656
```

Reply:
Why are we dropping these ids? I agree that they are odd but is the decision to drop them justified?

Reply:
The UniProt mapping of some of those ids clash with Ensembl ids (thanks @DSuveges):
```
H3BQV5	ENSG00000197943
H3BQV5	LRG_376
```
And they don't seem to be included in the Open Targets platform, so it makes sense to remove them.